recipes/giotto-learn/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ name }}/{{ name }}.git
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - git submodule update --init --recursive
```

Comment:
Why do you need these submodules?
is that a dependency?

Reply:
We are indeed using some external packages as dependencies and we need to compile them (in particular pybind11, ripser.py and a modified version of gudhi). We add them to the project as git submodules.

Reply:
The right way is to add your dependencies in the ``host`` section and use ``cmake`` to find them

Reply:
these dependencies are C++ code to be bound to python by pybind11, they are not standard dependencies such as sklearn or Numpy. pybind11 requires binding files (now in the folder externals/giotto/bindings/ ) and the actual scripts. In the pybind11 examples they also clone the repositories in order to bind them: <https://github.com/pybind/cmake_example>.

Reply:
Sorry, but ``pybind11`` is a header-only library and we have it already on ``conda-forge``

you can use on ``CMakeLists.txt`` something like
```
find_package(pybind11 REQUIRED)
```
You really do not need the ``pybind11` code to generate the bindings for python

Reply:
Please see
https://pybind11.readthedocs.io/en/stable/compiling.html#find-package-vs-add-subdirectory

> For CMake-based projects that don’t include the pybind11 repository internally, **an external installation can be detected through find_package(pybind11)**

Reply:
Done!! Thank you for the suggestion and the links!

Reply:
As you changed your ``CMakeLists.txt`` to find the ``pybind11`` installed, I think you do not need this line, right?
```suggestion
```